services/QuillLMS/client/app/bundles/Diagnostic/components/home.tsx

```diff
@@ -79,7 +79,8 @@ export const Home = () => {
           />
         </aside>}
         <main style={{ height: '100vh', overflow: 'auto' }}>
-          {isPlaying && !isTeacherOrAdmin && <button className="skip-main" onClick={handleSkipToMainContentClick} type="button">Skip to main content</button>}
+          <ScreenreaderInstructions />
+          <button className="skip-main" onClick={handleSkipToMainContentClick} type="button">Skip to main content</button>
```

Comment:
Are we removing the two nil checks on purpose here?

Reply:
Yes - I'm not totally sure why we were here initially but this is the only place in five similar files we had these nil checks in place, and I don't think it's necessary.